llama_cpp/llama.py

```diff
@@ -484,6 +497,7 @@ def generate(
         mirostat_mode: int = 0,
         mirostat_tau: float = 5.0,
         mirostat_eta: float = 0.1,
+        logits_processors=None
```

Comment:
Can we move `stopping_criteria` here so it matches the hugginface `generate` API? Then only use `generate` in the guidance PR.

Reply:
I would have to move the stop criteria check down here. Right now I'm using create completion in guidance.